recipes/poolsnp/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.0.0" %}
```

Comment:
```suggestion
{% set version = "0.1.0" %}
```
I'd suggest to start with 0.x since there is no official release of https://github.com/capoony/PoolSNP yet

Reply:
Updated version numbers